ember-cli-build.js

```diff
@@ -3,67 +3,49 @@
 const MergeTrees = require('broccoli-merge-trees');
 const Funnel = require('broccoli-funnel');
 const Rollup = require('broccoli-rollup');
-const Plugin = require('broccoli-plugin');
+const stew = require('broccoli-stew');
 const path = require('path');
-
-const walkSync = require('walk-sync');
-const fs = require('fs');
-const mkdirp = require('mkdirp').sync;
-
-class TestIndex extends Plugin {
-  constructor(input) {
-    super([input], {
-      persistentOutput: true,
-      annotation: 'tests/index.js'
-    });
+const typescript = require('broccoli-typescript-compiler');
+const ts = typescript(stew.find('lib'), {
```

Comment:
You can pass `lib` straight to `typescript`, let me also recommend using `__dirname + '/lib'` that way this broccoli pipeline can be used if `process.cwd()` is not the root of this project (if lets see ember is building)